guides/common/attributes.adoc

```diff
@@ -15,6 +15,7 @@
 :context: assembly
 
 ifeval::["{build}" == "foreman"]
+:Team: Foreman developers
```

Comment:
should add this in alphabetical order?

Reply:
I've fixed that already, just refresh. I've used GH feature of resolving conflicts and it still says "
This branch cannot be rebased due to conflicts" tho, need to do this locally.